merqube_client_lib/constants.py

```diff
@@ -17,3 +17,5 @@
 
 PRICE_RETURN: Final[str] = "price_return"
 TOTAL_RETURN: Final[str] = "total_return"
+
+INDEX_RUN_STATUS_METRIC: Final[str] = "index_run_status"
```

Comment:
Is this needed?

Reply:
dev and indexapi both independently redefine this, and both inherit from this package. idea is for them to import it from here